kernel/configs/fragments/x86_64/gpu/x86.conf

```diff
@@ -0,0 +1 @@
+CONFIG_PCI_MMCONFIG=y
```

Comment:
Can you comment why this is required?

Reply:
CONFIG_PCI_MMCONFIG: Support mmconfig PCI config space access. 
To enable the MMIO access method for PCIe device, we have to enable it.

Reply:
I removed unnecessary parameters.
Here are the minimum parameters required to compile and load the Nvidia GPU driver in a kata container:
```
# Support mmconfig PCI config space access.
# It's used to enable the MMIO access method for PCIe devices.
CONFIG_PCI_MMCONFIG=y

# Support to load driver module
CONFIG_MODULES=y
CONFIG_MODULE_UNLOAD=y
CONFIG_MODULE_SIG=y

# Linux kernel version suffix
CONFIG_LOCALVERSION="-gpu"
```

Reply:
Thanks @Jimmy-Xu